docs/dev.md

```diff
@@ -65,7 +54,7 @@ Edit `MAJOR` when there are breaking changes in the collection. Edit `MINOR` whe
 
 The collection has multiple CI pipelines that run on every PR:
 
-- PR pipeline: [`.github/workflows/pr.yml`](../.github/workflows/pr.yml) that runs the sanity tests and fails if there are any regression. It also runs a check on the PR Dependencies (`Depends-On:` lines in the commit message) to make sure that the dependencies are merged before the PR.
+- PR pipeline: [`.github/workflows/pr.yml`](../.github/workflows/pr.yml) that runs the sanity tests and Ansible lint. It fails if there are any regression. It also runs a check on the PR Dependencies (`Depends-On:` lines in the commit message) to make sure that the dependencies are merged before the PR.
```

Comment:
if there are any regression -> if there is any regression
(this is not related to this change but maybe it's a good time to fix it)

Reply:
done